set/set.go

```diff
@@ -0,0 +1,147 @@
+package set
+
+// GetSet gives new set.
+func GetSet(items ...interface{}) Set {
```

Comment:
This style of naming Go naming conventions. Go package names are enough to make it clear what you're getting and instead of creating a constructor like `set.GetSet` it would be much more natural to use it like `set.New(items)`. Don't you agree?

Reply:
Nice idea actually, we should conventions. I'll look into it.